build.gradle

```diff
@@ -70,7 +70,7 @@ ext {
     hsqldbVersion = '2.7.4'
     h2Version = '2.3.232'
     jacksonVersion = '2.19.2'
-    jackson3Version = '3.0.0-rc5'
+    jackson3Version = '3.0.0-rc6'
```

Comment:
Please, hold of any rc6 changes.
See my comment about Joda.
I am from the phone , so cannot be very helpful on weekend 😆